apps/aggiemap/src/app/modules/services/trip-planner/core/trip-planner-core.ts

```diff
@@ -296,10 +296,10 @@ export class TripPoint {
    * @memberof TripPoint
    */
   public toEsriGraphic(): esri.Graphic {
-    const attributes: any = { ...this.attributes };
-    const geometry: any = { ...this.originGeometry.raw };
+    const attributes = { ...this.attributes };
+    const geometry: esri.Geometry = this.originGeometry.raw.clone();
```

Comment:
`origin.geometry.raw` is not a full Geometry class instance, and does not have .clone().

This breaks search selection and trip planner selection from search suggestions. 

Shallow copy of raw geometry necessary here.